src/daemon.rs

```diff
@@ -306,9 +310,9 @@ fn sock_send(command: &str) -> std::io::Result<()> {
 fn send_command(hotkey: config::Hotkey) {
     log::info!("Hotkey pressed: {:#?}", hotkey);
     if let Err(e) = sock_send(&hotkey.command) {
-        log::error!("Failed to send command over IPC.");
-        log::error!("Is swhks running?");
-        log::error!("{:#?}", e)
+        log::error!("Failed to send command to swhks through IPC.");
+        log::error!("Please make sure that swhks is running.");
+        log::error!("Error code: {:#?}", e)
```

Comment:
it's not necessarily an error code, I think `Err: {:#?}", e` will be better here and that's the convention we've used all over the codebase.